debian/control

```diff
@@ -0,0 +1,41 @@
+Source: freedom-maker
+Maintainer: Joseph Nuthalapati <njoseph@riseup.net>
```

Comment:
You can set the Maintainer as "FreedomBox packaging team <freedombox-pkg-team@lists.alioth.debian.org>". Then add anyone who might upload as an uploader.